pkg/core/resources/model/rest/unmarshaller.go

```diff
@@ -62,6 +62,13 @@ func (u *unmarshaler) Unmarshal(bytes []byte) (Resource, error) {
 	if err := u.unmarshalFn(bytes, restResource); err != nil {
 		return nil, errors.Wrapf(err, "invalid %s object %q", meta.Type, meta.Name)
 	}
+
+	if rv, ok := resource.(core_model.ResourceValidator); ok {
```

Comment:
note: this will trigger validation on old policies as well, are we ok with that @jakubdyszkiewicz @lobkovilya ?

Reply:
I think it should be ok. One consequence of doing so is that we will do more client-side validation because this method is called on `kumactl apply -f -`